src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label>
+            Surname:
+            <input
+             name="surname"
+             type="text"
+             autocomplete="off"
+            >
+          </label>
+          <label>
+            Name:
+            <input type="text"
```

Comment:
Please format your code, check for all cases!
![image](https://user-images.githubusercontent.com/54936044/228381140-9b02b221-f458-491b-882c-5d03d08e24c4.png)